extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts

```diff
@@ -49,9 +162,11 @@ export class PgpBlockViewRenderModule {
       await AcctStore.set(this.view.acctEmail, { successfully_received_at_leat_one_message: true });
     }
     if (!isErr) { // rendering message content
+      $('.pgp_print_button').show();
       const pgpBlock = $('#pgp_block').html(Xss.htmlSanitizeKeepBasicTags(htmlContent, 'IMG-TO-LINK')); // xss-sanitized
       pgpBlock.find('a.image_src_link').one('click', this.view.setHandler((el, ev) => this.displayImageSrcLinkAsImg(el as HTMLAnchorElement, ev as JQuery.Event<HTMLAnchorElement, null>)));
     } else { // rendering our own ui
+      $('.pgp_print_button').hide();
```

Comment:
Do we need this statement? Is there really a scenario when we first render the message content and then overwrite it with an error?

Reply:
Yes. Like when passphrase is expired(or not exist) or when user taps `end passphrase` section

Reply:
Meaning, when passphrase isn't known, we have `Print` button? I don't quite understand.

Reply:
Sorry I misunderstood what you mean.
Yeah. We indeed don't need that hide statement as initial print button visibility is hidden.